tests/Behat/TraversingContext.php

```diff
@@ -30,10 +32,16 @@ public function clickLastLink(string $link): void
     }
 
     /**
-     * @When I wait for the modal to load
+     * @When I wait for the modal :modal to load
      */
-    public function iWaitForTheModalToLoad(): void
+    public function iWaitForTheModalToLoad(string $modal): void
```

Comment:
why are you talking about modals here ? imho you can skip this work and make it generic, as you are only waiting for a selector